HVCAENx527App/Db/HVCAENx527gbl.db

```diff
@@ -168,3 +168,19 @@ record (stringin, "$(P):CHANLIST:DELIM")
 {
 	field(VAL, " ")
 }
+
+## record how many time event 11 (EVNTNO_T1) is posted
+record(calc, "$(P):EVNTO_T1:COUNT")
+{
+    field(SCAN, "Event")
+    field(EVNT, "11")
+    field(CALC, "A:=A+1;A")
+}
```

Comment:
It feels like it would be useful to add this to the inst archive so that we can check it later